app/next-client-app/api/concepts.ts

```diff
@@ -0,0 +1,56 @@
+"use server";
+import request from "@/lib/api/request";
+import { revalidatePath } from "next/cache";
+
+const fetchKeys = {
+  conceptFilter: (filter: string) =>
+    `omop/conceptsfilter/?concept_id__in=${filter}`,
+  addConcept: "v2/scanreportconcept/",
+  deleteConcept: (conceptId: number) => `scanreportconcepts/${conceptId}`,
+  scanreportConcepts: (filter?: string) =>
+    `scanreportconceptsfilter/?${filter}`,
+};
+
+export async function getScanReportConcepts(
+  filter: string
+): Promise<ScanReportConcept[]> {
+  try {
+    return await request<ScanReportConcept[]>(
+      fetchKeys.scanreportConcepts(filter)
+    );
+  } catch (error) {
+    console.warn("Failed to fetch data.");
+    return [];
+  }
+}
+
+export async function getConceptFilters(filter: string): Promise<Concept[]> {
```

Comment:
Is this still needed? If not delete it.

Reply:
This function will contribute to mapping the newly added concept with the scan report field. So I think it is useful to keep it